browser/ui/webui/brave_web_ui_controller_factory.cc

```diff
@@ -47,6 +53,8 @@ WebUIFactoryFunction GetWebUIFactoryFunction(WebUI* web_ui,
   if (url.host_piece() == kPaymentsHost ||
       url.host_piece() ==  chrome::kChromeUINewTabHost) {
     return &NewWebUI<BasicUI>;
+  } else if (url.spec() == kWelcomeRemoteURL) {
```

Comment:
@diracdeltas @bbondy - this isn't directly related to this PR, but I think a remote welcome url is a privacy issue. Is there any reason why it can't be an internal page?

Reply:
We discussed changing to local welcome page but no one has had time to do that yet. Note that browser-laptop is remote too so we have at least baseline parity with that.  But we did talk about wanting to move to a local one.

Reply:
seems like it should be pretty easy to do now that we have a webui page for it. Is there a ticket or should I create one?

Reply:
I thought there was but I couldn't find one, so I posted it here:
https://github.com/brave/brave-browser/issues/276

Reply:
Thanks!

Reply:
here's the original issue by @flamsmark https://github.com/brave/browser-laptop/issues/12691. i also feel strongly that this should be baked into brave for privacy reasons.